src/platform/web/theming/shared/color.mjs

```diff
@@ -13,10 +13,10 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
+import pkg from 'off-color';
+const offColor = pkg.offColor;
```

Comment:
are you doing a default import because this is a commonjs package? No way to import only what we need I suppose?

Reply:
Yep, `yarn test` fails otherwise.